dotcom-rendering/src/web/components/ArticleMeta.tsx

```diff
@@ -260,10 +260,18 @@ export const ArticleMeta = ({
 	const bylineImageUrl = getBylineImageUrl(tags);
 	const authorName = getAuthorName(tags);
 
+	const showAvatarFromAuthor = () => {
+		if (!('byline' in author) || author.byline === '') {
```

Comment:
is this the same as `(!author.byline || author.byline === '')` - i find this consistency of checking for attrs a bit better/more readable if it evaluates to the same thing